apps/dotcms-block-editor/src/app/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -31,6 +31,26 @@ dotcms-block-editor {
         min-height: 100%;
         outline: none;
         padding: $spacing-5 $spacing-8;
+
+        pre {
+            background: #0d0d0d;
+            border-radius: 0.5rem;
+            color: #fff;
+            font-family: "JetBrainsMono", monospace;
```

Comment:
I don't think a lot of people have that font installed, according to [this post](https://www.digitalocean.com/community/tutorials/css-system-font-stack) we should use

```
code {
  font-family: SFMono-Regular, Menlo, Monaco,
               Consolas, "Liberation Mono",
               "Courier New", monospace;
}
```